### PR TITLE
Update to Async `Keyboard`

### PR DESCRIPTION
**Pull request by inancgumus**
## What?

Update relevant `Keyboard` methods from sync to async.

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [ ] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

## Related PR(s)/Issue(s)

Updates: grafana/xk6-browser#1304

Closes: grafana/xk6-browser#765

### CODE REVIEW

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
 
 # press(key[, options])
+
+Dispatches a `keydown` event followed by an `keyup` event. This is useful for simulating pressing and releasing a key.
+
+| Parameter     | Type   | Default | Description                                                     |
+| ------------- | ------ | ------- | --------------------------------------------------------------- |
+| key           | string | `''`    | Name of the key to press, such as `Enter`.                      |
+| options       | object | `null`  | Optional settings.                                              |
+| options.delay | number | `0`     | Optional. Time to wait before pressing the key in milliseconds. |
```
Comment (ankur22) [inline docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md:14]: Is this the time to wait between `keyDown` and `keyUp`?

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
 ---
 title: 'press(key[, options])'
-description: 'TODO'
+description: 'Dispatches a `keydown` event followed by an `keyup` event. This is useful for simulating pressing and releasing a key.'
```
Comment (ankur22) [inline docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md:3]:
```suggestion
description: 'Dispatches a `keydown` event followed by a `keyup` event. This is useful for simulating pressing and releasing a key.'
```

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
 ---
 
 # press(key[, options])
+
+Dispatches a `keydown` event followed by an `keyup` event. This is useful for simulating pressing and releasing a key.
```
Comment (ankur22) [inline docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md:8]:
```suggestion
Dispatches a `keydown` event followed by a `keyup` event. This is useful for simulating pressing and releasing a key.
```

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/type.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
 
 # type(text[, options])
+
+Dispatches a `keydown`, `keypress` or`input`, and `keyup` events for each character in the text. This is useful for simulating typing text into an input field.
+
+| Parameter     | Type   | Default | Description                                                    |
+| ------------- | ------ | ------- | -------------------------------------------------------------- |
+| text          | string | `''`    | The text to type into the input field.                         |
+| options       | object | `null`  | Optional settings.                                             |
+| options.delay | number | `0`     | Optional. Time to wait before typing the text in milliseconds. |
```
Comment (ankur22) [inline docs/sources/next/javascript-api/k6-experimental/browser/keyboard/type.md:14]: Or is this the delay in between each key press?

**Comment by inancgumus** [comment]
> What about examples for each of the APIs?

We're on a tight schedule and I prefer adding them later. I've created an issue: https://github.com/grafana/k6-docs/issues/1597.

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md (hunk 1 above)
Comment (inancgumus) [reply docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md:14]: It's the time before all those.

https://github.com/grafana/xk6-browser/blob/99686dfc84e86e0d02be70323a5cd4902c306438/common/keyboard.go#L243-L247

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/type.md (hunk 4 above)
Comment (inancgumus) [reply docs/sources/next/javascript-api/k6-experimental/browser/keyboard/type.md]: See this: https://github.com/grafana/k6-docs/pull/1592/commits/e6dc7ce3f10f01b8bb2519a0f2fcfd6c5b4e7dbe

File: docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md (hunk 1 above)
Comment (inancgumus) [reply docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md]: Changed this here: [5ce2ca6](https://github.com/grafana/k6-docs/pull/1592/commits/5ce2ca67e8ae6f4b41fdee7621c6bd8ce7a61576)

**Pull request merged**